src/main/webapp/WEB-INF/web.xml

```diff
@@ -115,6 +125,16 @@
         <url-pattern>/*</url-pattern>
     </filter-mapping>
 
+    <filter-mapping>
+        <filter-name>cloudfrontForwardedHeaderFilter</filter-name>
+        <url-pattern>/*</url-pattern>
+    </filter-mapping>
+
+    <filter-mapping>
```

Comment:
Is it possible to make these filters optional? That can only be turned on with a flag in the Studio configuration? Because it could be an issue, if the load balancer or reverse proxy doesn't overwrite the flags, or a client is using no reverse proxy.